build.py

```diff
@@ -641,6 +643,41 @@ def build(self):
 
 Project.add(Project_glib_openssl())
 
+class Project_graphene(GitRepo, Project):

```

Comment:
I am ok with this as the first patch but since the meson projects will be the same I'd really like a Meson class inheriting from Project and then the graphene project inheriting from Meson